frontend/src/App.css

```diff
@@ -17,6 +17,16 @@
   display: block;
   margin-left: auto;
   margin-right: auto;
-  width: 25%;
-  height: 25%;
+  width: 250px;
+  height: 250px;
+}
+
+@media only screen and (max-width: 600px) {
+  .logo {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
```

Comment:
this doesn't change: you don't have to write it a second time

Reply:
✅